package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "phone_directory",
+  "version": "1.0.0",
+  "description": "a simple api to find all the phone number",
+  "main": "index.js",
+  "type": "module",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "nodemon src/index.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/sathyarjunab/node-phone-directory.git"
+  },
+  "author": "sathyarjunab",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/sathyarjunab/node-phone-directory/issues"
+  },
+  "homepage": "https://github.com/sathyarjunab/node-phone-directory#readme",
+  "devDependencies": {
+    "body-parser": "^1.20.2",
+    "express": "^4.19.2",
+    "mongoose": "^8.4.1",
+    "nodemon": "^3.1.3",
```

Comment:
only nodemon is dev dependency rest other are normal dependency 
check what's dev and normal dependency online

Reply:
done.